runtime/src/verify.rs

```diff
@@ -1,25 +1,15 @@
 // Licensed under the Apache-2.0 license
 
-use crate::{CommandId, Drivers, EcdsaVerifyCmd};
+use crate::{Drivers, MailboxResp, EcdsaVerifyReq};
 use caliptra_drivers::{
     Array4x12, CaliptraError, CaliptraResult, Ecc384PubKey, Ecc384Result, Ecc384Scalar,
     Ecc384Signature,
 };
 use zerocopy::FromBytes;
 
-/// Start of payload (skips checksum field).
-const PAYLOAD_OFFSET: usize = 4;
-
 /// Handle the `ECDSA384_SIGNATURE_VERIFY` mailbox command
-pub(crate) fn handle_ecdsa_verify(drivers: &mut Drivers, cmd_args: &[u8]) -> CaliptraResult<()> {
-    if let Some(cmd) = EcdsaVerifyCmd::read_from(cmd_args) {
-        if !caliptra_common::checksum::verify_checksum(
-            cmd.chksum,
-            CommandId::ECDSA384_VERIFY.into(),
-            &cmd_args[PAYLOAD_OFFSET..],
-        ) {
-            return Err(CaliptraError::RUNTIME_INVALID_CHECKSUM);
-        }
+pub(crate) fn handle_ecdsa_verify(drivers: &mut Drivers, cmd_args: &[u8]) -> CaliptraResult<MailboxResp> {
```

Comment:
The handler has  low cohesion with  this file.

Reply:
Let me know if the new format addresses this at all. I didn't catch you on IM before I made this most recent change.